packages/component-library-react/src/Calendar/Calendar.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { formatISO } from 'date-fns';
+import { nl } from 'date-fns/locale';
+import { Calendar, Events } from './index';
+
+describe('Calendar', () => {
+  it('invokes an onclick callback with a selected date', () => {
+    let selectedDate = '';
+    render(
+      <Calendar
+        currentDate={new Date(2023, 5, 11)}
+        onCalendarClick={(date) => {
+          selectedDate = date;
+        }}
+        locale={nl}
+      />,
+    );
+
+    let dayButton = screen.getByText('15');
```

Comment:
To include accessibility in our tests I do prefer [`getByRole`](https://testing-library.com/docs/queries/byrole) with the `name` option which selects the aria-label used.

So for this dayButton:

`screen.getByRole('button', {name: 'maandag 15 mei 2023'})`

Reply:
Nice tip thanks!